src/features/user/MFV2/ContributionsMap/MyForestV2.module.scss

```diff
@@ -0,0 +1,223 @@
+.mapCreditsContainer {
+  position: absolute;
+  right: 48px;
+  bottom: 2px;
+  z-index: 1;
+  cursor: pointer;
+}
+
+.mapCredit {
+  padding: 10px;
+  width: 330px;
+}
+.contributionPopup,
+.registeredTreePopup {
+  > div {
+    border-radius: 8px;
+  }
+}
+
+.contributionPopup {
+  > div:nth-child(1) {
+    display: none;
+  }
+}
+
+.contributionPopup {
+  > div {
+    padding: 0px;
+  }
+}
+
+.customDonateButton {
+  padding: 5px 16px;
+  font-size: 12px;
+  white-space: wrap;
+}
```

Comment:
Not sure this needed anymore, as the donate button in the popup is the standard tertiary WebappButton.